addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/handler/GlobalCacheHandler.java

```diff
@@ -264,7 +272,7 @@ private String lookupCode(Command command, ChannelUID channelUID) {
                 code = convertHexToGC(code);
                 logger.debug("Converted hex code is: {}", code);
             } catch (HexCodeConversionException e) {
-                logger.warn("Failed to convert hex code to GC format");
+                logger.info("Failed to convert hex code to GC format");
```

Comment:
ok... This whole binding is anyhow pretty exceptional in terms of "low level functionality" - normally, a user should never be required to check the logs if the issue is within his configuration. :-)